src/pushsource/_impl/backend/errata_source/errata_source.py

```diff
@@ -247,6 +251,22 @@ def _push_items_from_container_manifests(self, erratum, docker_file_list):
 
         return out
 
+    def _push_items_from_appliance_image_list(self, appliance_image_list):
+        if not appliance_image_list:
+            return []
+
+        nvr_list = []
+        out = []
+        for image in appliance_image_list:
+            for nvr in image.keys():
+                nvr_list.append(nvr)
```

Comment:
Just a suggestion to simplify this:

```suggestion
        for image in appliance_image_list:
            nvr_list.extend([nvr for nvr in image.keys()])
```

Reply:
updated